anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -770,202 +848,87 @@ def _run_grypedb_sync_task(self, checksum: str, grype_db_data: bytes) -> None:
                 f.write(grype_db_data)
             GrypeDBSyncManager.run_grypedb_sync(grypedb_file.path)
 
-    def _sync_group(
+    def _process_group_file_records(
         self,
+        db: Session,
         group_download_result: GroupDownloadResult,
-        full_flush: bool = False,
-        local_repo: Optional[LocalFeedDataRepo] = None,
-        operation_id: Optional[str] = None,
-    ):
+        group_obj: FeedGroupMetadata,
+        local_repo: Optional[LocalFeedDataRepo],
+    ) -> int:
         """
-        Sync data from a single group and return the data. This operation is scoped to a transaction on the db.
-
+        Convert the download results for a feed group into database records and write to the database session
+        Does not flush, so transaction can be rolled back if an exception is encountered.
```

Comment:
Same comment as above, if we can reference where that happens I think it makes it easier for the next person reading through this to understand the flow.

Reply:
docstring corrected